content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -217,7 +217,7 @@ export class _DiscoveryStreamBase extends React.PureComponent {
     const topSites = extractComponent("TopSites");
     const message = extractComponent("Message") || {
       header: {
-        link_text: topStories.learnMore.link.id,
+        link_id: topStories.learnMore.link.id,
```

Comment:
A DSMessage component only has link_text and not link_id, so this isn't quite right, but I'll fix it up with my getFormattedMessage change